src/core/common/ResourseTemplates/ListTemplate.js

```diff
@@ -146,6 +143,16 @@ class ListTemplate extends Component {
         })
     };
 
+    /**
+     * This function check is current page for list
+     *
+     * @author Bogdan Shcherban <bsc@piogroup.net>
+     * @return {boolean}
+     */
+    isListPage = () => {
```

Comment:
I find this method strange. this is a ListTemplate, where else could it be used than a List page?

Reply:
Could you please look at the page behavior:
https://drive.google.com/file/d/1HpP4HCdp8SSaQQl7KSG94QxmFTx6eSFq/view?usp=sharing

ListTemplate is used not only at page with table. It also used at:
1) creape page:
https://apple-cake.ripple.foundation/#/medications/create
2) show / edit page
https://apple-cake.ripple.foundation/#/medications/ethercis-14b82e54-86f3-42d2-b7a9-5ce1d296e828

This pages have different URLs. We should know, is current page is List-page, to set Gris-parameters.
```
<Grid className={classes.list} item xs={12} sm={this.isListPage() ? 12 : 6}>.....</Grid>
```

Why we check it?
List-page table is located in 12 column as in desktop, as in mobile mode. 
At create/show/edit-page table should use 6 columns at desktop screen and 12 columns in mobile mode.

Reply:
OK,  understood. You should probably add a comment in the code to explain that, too.

Reply:
Comment was added.